src/main/java/duke/application/MainWindow.java

```diff
@@ -0,0 +1,77 @@
+package duke.application;
+
+import duke.io.Parser;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/UserIcon.png"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/DukeIcon.png"));
+
+    private Parser parser;
+
+    /**
+     * Initialise settings, and greet user
```

Comment:
A few of these Javadoc's headers for this method should have "s", such as initializes, greets, etc.